app/script/client/ClientError.js

```diff
@@ -37,18 +37,15 @@ z.client.ClientError = class ClientError extends Error {
       case z.client.ClientError.TYPE.DATABASE_FAILURE:
         this.message = 'Client related database transaction failed';
         break;
-      case z.client.ClientError.TYPE.MISSING_ON_BACKEND:
-        this.message = 'Local client does not exist on backend';
-        break;
       case z.client.ClientError.TYPE.NO_CLIENT_ID:
         this.message = 'Client ID is not defined';
         break;
-      case z.client.ClientError.TYPE.NO_LOCAL_CLIENT:
-        this.message = 'No local client found in database';
-        break;
       case z.client.ClientError.TYPE.NO_USER_ID:
         this.message = 'User ID is not defined';
         break;
+      case z.client.ClientError.TYPE.NO_VALID_CLIENT:
+        this.message = 'No valid client found in locally';
```

Comment:
`'No valid client found in local database'` or `'No valid client found locally'`